code/main.py

```diff
@@ -90,7 +90,8 @@ def main():
         validate=client.deployments.validate(resource_group,"azure-sample",deployment_properties)
```

Comment:
please change this deployment name to 
github repository name from which deployment is being made.

Reply:
updated